apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/utils/TimeUtils.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.apollocurrency.aplwallet.apl.core.utils;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+/**
+ *
+ * @author Serhiy Lymar
+ */
+public class TimeUtils {
+    
+    public static String millisToDate(long millis){
+        return DateFormat.getDateInstance(DateFormat.LONG).format(millis);    
+    }
+    
+    
+    public static long dateToMillis (String dateString ){ 
```

Comment:
Need to add private constructor and move format to the constants. Also better to use Java 8 Time api.